src/main/java/com/exasol/ExaMetadata.java

```diff
@@ -3,206 +3,296 @@
-     * @param column id of the column, starting with 0
-     * @return sql type of the specified output column.
-     *         This is a string in SQL syntax, e.g. "DECIMAL(18,0)"
+     * Get the SQL data type of the output column at the given index.
+     * <p>
+     * This is a string in SQL syntax, e.g. {@code DECIMAL(18,0)}.
+     * </p>
+     *
+     * @param column index of the column, starting with 0
+     *
+     * @return sql type of the specified output column
      */
     public String getOutputColumnSqlType(int column) throws ExaIterationException;
 
     /**
-     * @param column id of the column, starting with 0
-     * @return data type precision of the specified output column, e.g.
-     *         the precision of a DECIMAL data type
+     * Get the precision of the output column at the given index.
+     *
+     * @param column index of the column, starting with 0
+     *
+     * @return data type precision of the specified output column, e.g. the precision of a {@code DECIMAL} data type
      */
     public long getOutputColumnPrecision(int column) throws ExaIterationException;
 
     /**
-     * @param column id of the column, starting with 0
-     * @return data type scale of the specified output column, e.g.
-     *         the scale of a DECIMAL data type
+     * Get the scale of the column at the given index.
+     *
+     * @param column index of the column, starting with 0
+     *
+     * @return data type scale of the specified output column, e.g. the scale of a {@code DECIMAL} data type
      */
     public long getOutputColumnScale(int column) throws ExaIterationException;
 
     /**
-     * @param column id of the column, starting with 0
-     * @return data type length of the specified column, e.g. the length of a VARCHAR data type
+     * Get the length of the output column at the given index.
+     *
+     * @param column index of the column, starting with 0
+     *
+     * @return data type length of the specified column, e.g. the length of a {@code VARCHAR} data type
      */
     public long getOutputColumnLength(int column) throws ExaIterationException;
 
     /**
-     * Dynamically loads the code from the specified script, compiles it, and
-     * returns an instance of the main script class. You can use Java Reflection
-     * to work with this class.
-     *
-     * Please note that there is a simple way to include other code using
-     * the keywords %import and %jar in the script code (see user manual).
-     *
+     * Dynamically loads the code from the specified script, compiles it, and returns an instance of the main script
+     * class.
+     * <p>
+     * You can use Java Reflection to work with this class.
+     * </p>
+     * <p>
+     * Please note that there is a simple way to include other code using the keywords {@code %import} and {@code %jar}
+     * in the script code (see user manual).
+     * </p>
+     * 
      * @param name The name of the script to be imported (context-sensitive)
```

Comment:
```suggestion
     * @param name the name of the script to be imported (context-sensitive)
```
What does "context-sensitive" mean? Maybe a typo -> "case-sensitive"?